content/en/docs/setup/independent/high-availability.md

```diff
@@ -201,63 +215,40 @@ SSH is required if you want to control all nodes from a single machine.
     kubectl get pod -n kube-system -w
     ```
 
-    - It's recommended that you join new control plane nodes only after the first node has finished initializing.
+    - It's recommended that you join new control plane nodes in a sequence, only after the first node has finished initializing.
 
-1.  Copy the certificate files from the first control plane node to the rest:
+1.  Optionally copy the `admin.conf` file from the first control plane node to the rest:
```

Comment:
Why we should copy this config? [1]

Reply:
to use kubectl on the other CP nodes.
i'm OK with removing this but waiting on review by @fabriziopandini

Reply:
The join don't generate a config?

Reply:
that's actually a good point...
possibly best to remove this anyway.

@Klaven please remove the lines `1.  Optionally copy the `admin......` and `Optionally move the `admin.conf` file created....`

Reply:
It is not necessary to copy the `admin.conf` file
However, I think we should still provide user instruction for manually copy certs if they want to opt out from the `--experimental-upload-certs` feature, so my suggestion is to move everyting related to "configure SSH" and scp certs (without the `admin.conf` file) in a dedicated paragraph "manual certs distribution"